page-submit.php

```diff
@@ -0,0 +1,63 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section class="open-submissions">
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $open_submissions = IGV_get_option('_igv_submit_options', '_igv_open_submissions');
+
+    if (!empty($open_submissions)) {
+      foreach($open_submissions as $index => $submission) {
+  ?>
+  <article <?php post_class('container submission'); ?>>
+    <div class="grid-row margin-top-basic margin-bottom-basic u-flex-center">
+      <div class="grid-item item-s-6">
+        <?php echo wp_get_attachment_image($submission['_igv_submission_image_id'], 'l-6'); ?>
+      </div>
+    </div>
+    <div class="grid-row u-flex-center">
+      <div class="grid-item item-s-9 margin-bottom-basic text-align-center">
+        <h3><?php echo $submission['_igv_submission_title']; ?></h3>
+      </div>
+    </div>
+    <div class="grid-row u-flex-center">
+      <div class="grid-item item-s-8 margin-bottom-basic">
+        <?php echo apply_filters('the_content', $submission['_igv_submission_desc']); ?>
+      </div>
+    </div>
+    <div class="grid-row u-flex-center">
+      <div class="grid-item item-s-8 margin-bottom-basic">
+      <div class="text-align-center"><a class="expandable-toggle no-hide link-button" data-exapandable-id="expandable-<?php echo $index; ?>"><?php echo $submission['_igv_submission_button_text']; ?></a></div>
+      </div>
+    </div>
+    <div id="expandable-<?php echo $index; ?>" class="expandable-content grid-row u-flex-center">
```

Comment:
Basically to do with what the toggling item does or does not. Could build onto the existing codebase but the current implementation smells bad, just looks like backwards hacking to fix a different usecase